site/content/docs/commands/deploy.en.md

```diff
@@ -82,3 +82,19 @@ Initializes and deploys a service named "backend" to a "prod" environment.
 ```console
 $ copilot deploy --init-wkld --deploy-env=false --env prod --name backend
 ```
+
+Deploys all local, initialized workloads in no particular order.
+```console
+$ copilot deploy --all --env prod --name backend
+```
+
+Deploys multiple workloads in a prescribed order (fe and worker, then be).
+```console
+$ copilot deploy -n fe/1 -n be/2 -n worker/1
+```
+
+Initializes and deploys all local workloads after deploying environment changes.
```

Comment:
What do you think about this? Because it could be the initial deployment....
```suggestion
Initializes and deploys all local workloads after (re)deploying the `prod` environment.
```